lib/nexpose/common.rb

```diff
@@ -135,17 +135,12 @@ class Schedule < APIObject
     attr_accessor :type
     # The repeat interval based upon type.
     attr_accessor :interval
-    # The earliest date to generate the report on (in ISO 8601 format).
+    # Starting time of the scheduled scan (in ISO 8601 format).
     attr_accessor :start
-
     # The amount of time, in minutes, to allow execution before stopping.
     attr_accessor :max_duration
     # The date after which the schedule is disabled, in ISO 8601 format.
     attr_accessor :not_valid_after
-
-    # TODO: Remove this unused attribute
-    attr_accessor :incremental
```

Comment:
No.
Also, if we're removing that attr, then we should remove the other places in the code that are calling it. [schedule.incremental = (xml.attributes['incremental'] && xml.attributes['incremental'] == '1'))](https://github.com/rapid7/nexpose-client/blob/905a1186a3c7d423c67005d52cc2d45e0583b0ce/lib/nexpose/common.rb#L273)

@dmurray-r7 @gschneider-r7